Makefile

```diff
@@ -0,0 +1,31 @@
+# define the name of the virtual environment directory
+VENV := venv
+
+# default target, when make executed without arguments
```

Comment:
If I simply run `make`, it will just activate the local env?

Reply:
Yes, proof:
<img width="692" alt="image" src="https://user-images.githubusercontent.com/1254188/201914856-76075527-6af3-4cef-8710-e458236135f2.png">